rewrite-yaml/src/main/java/org/openrewrite/yaml/MergeYaml.java

```diff
@@ -58,12 +65,30 @@ public class MergeYaml extends Recipe {
     @Nullable
     String filePattern;
 
-    @Option(displayName = "Insert before property",
-            description = "Choose an insertion point when multiple mappings exist. Takes the `key` JsonPath into account.",
+    @Option(displayName = "Insert mode",
+            description = "Choose an insertion point when multiple mappings exist. Default is `Last`.",
+            valid = {"Before", "After", "Last"},
+            required = false,
+            example = "Last")
```

Comment:
I think we can at least drop the `example` here, as that's already implied by `valid`.
```suggestion
            valid = {"Before", "After", "Last"},
            required = false)
```
For enums we _might_ be able to drop the `valid` as well, but I haven't checked that. Could be good not to repeat ourselves.